### PR TITLE
Fix a bunch of issues with service API docs

### PR DESCRIPTION
**Pull request by cpuguy83**

### CODE REVIEW

**Comment by thaJeztah** [comment]
LGTM, but I'm on my phone so unable to check rendered output

File: docs/reference/api/docker_remote_api_v1.25.md [hunk 1]
```diff
@@ -3970,18 +3970,17 @@ JSON Parameters:
...
-            - **Propagation** – A propagation mode with the value `[r]private`, `[r]shared`, or
-              `[r]slave` (`bind` type mounts only).
-            - **MCSAccessMode** – MCS label for sharing mode (`bind` type mounts only).
+            - **Source** – Mount source (e.g. a volume name, a host path).
+            - **Type** – The mount type (`bind`, or `volume`).
             - **Writable** – A boolean indicating whether the mount should be writable.
-            - **VolumeTemplate** – Optional configuration for the volume.
-                - **Annotations** – User-defined name and labels for the volume.
-                - **Driver** – Name of the driver to be used and driver-specific options.
+            - **BindOptions** - Optional configuration for the `bind` type
```
Comment (avsm) [inline docs/reference/api/docker_remote_api_v1.25.md]: need a full stop at end of this for consistency with rest?

File: docs/reference/api/docker_remote_api_v1.25.md [hunk 2]
```diff
@@ -4188,18 +4187,17 @@ Update the service `id`.
...
+              - **Propagation** – A propagation mode with the value `[r]private`, `[r]shared`, or `[r]slave`.
+            - **VolumeOptions** – Optional configuration for the `volume` type.
+                - **Populate** – A boolean indicating if volume should be populated with the data from the target. (Default false)
+                - **Labels** – User-defined name and labels for the volume.
+                - **DriverConfig** – Map of driver-specific options.
+                  - **Name** - Name of the driver to use to create the volume
+                  - **Options** - key/value map of driver specific options
```
Comment (avsm) [inline docs/reference/api/docker_remote_api_v1.25.md]: These two also missing full stops.

Comment (cpuguy83) [inline docs/reference/api/docker_remote_api_v1.25.md]: Thank, updated, and a few more of these as well.

**Comment by thaJeztah** [comment]
removing cherry pick, because these changes (for 1.12) should be on the 1.24 API version

**Pull request merged**